apps/modernization-ui/src/apps/patient/add/extended/inputs/administrative/Administrative.tsx

```diff
@@ -0,0 +1,43 @@
+import { internalizeDate } from 'date';
+import { AdministrativeEntryFields } from 'apps/patient/profile/administrative/AdministrativeEntryFields';
+import { AdministrativeEntry } from 'apps/patient/profile/administrative/AdministrativeEntry';
```

Comment:
You're going to want to use this [AdministrativeEntry](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-ui/src/apps/patient/data/entry.ts#L11) that was created for the new form object.  This also means that we cannot resuse the `AdministrativeEntryFields` component from Patient Profile.  The main reason being the ability to support `Selectable`